content/news/2020-01-27.md

```diff
@@ -0,0 +1,41 @@
+---
+path: '/01-27-2020/spring-cloud-dataflow-2.4.0.M1-released'
+title: 'Spring Cloud Data Flow 2.4.0 M1 Released'
```

Comment:
Since now 2.4 GA is out, we don't need milestones. We can be selective about what to show.

Reply:
For instance, 2.5 is the latest master where we can highlight the milestones until when GA goes live.